<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetUp</class>
 <widget class="QWidget" name="SetUp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>1200</width>
    <height>900</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>基本信息管理</string>
  </property>
  <widget class="QPushButton" name="SaveButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>121</width>
     <height>45</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>121</width>
     <height>45</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>121</width>
     <height>45</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border: 2px solid #f3f3f3;/*设置线宽*/
	background-color: rgb(237, 242, 255);/*背景颜色*/
    border-radius: 15px;/*圆角*/
    padding: 2px 8px 15px 2px;  /*针对于组合框中的文本内容*/
	text-align:bottom;
    min-width: 9em;   /*# 组合框的最小宽度*/
    /*min-height: 5em;*/

	border-style:solid;/*边框为实线型*/
	border-width:2px;/*边框宽度*/
	border-color:rgb(77, 123, 255);/*边框颜色*/

	padding-left: 10px;/*左侧边距*/
}

QPushButton:hover
{
		color:#f3f3f3;
	background-color:rgb(102,178,255); /*改变背景色*/
	border-style:inset;/*改变边框风格*/
	padding-left:6px;
	padding-top:6px;
}</string>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </widget>
  <widget class="QLabel" name="bg_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>600</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>900</width>
     <height>600</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>900</width>
     <height>600</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>10</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>181</width>
     <height>31</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>181</width>
     <height>31</height>
    </size>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:700; color:#00aaff;&quot;&gt;基本信息管理&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="GPButton">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>10</y>
     <width>121</width>
     <height>45</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>121</width>
     <height>45</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>121</width>
     <height>45</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border: 2px solid #f3f3f3;/*设置线宽*/
	background-color: rgb(237, 242, 255);/*背景颜色*/
    border-radius: 15px;/*圆角*/
    padding: 2px 8px 15px 2px;  /*针对于组合框中的文本内容*/
	text-align:bottom;
    min-width: 9em;   /*# 组合框的最小宽度*/
    /*min-height: 5em;*/

	border-style:solid;/*边框为实线型*/
	border-width:2px;/*边框宽度*/
	border-color:rgb(77, 123, 255);/*边框颜色*/

	padding-left: 10px;/*左侧边距*/
}

QPushButton:hover
{
		color:#f3f3f3;
	background-color:rgb(102,178,255); /*改变背景色*/
	border-style:inset;/*改变边框风格*/
	padding-left:6px;
	padding-top:6px;
}</string>
   </property>
   <property name="text">
    <string>获取光谱</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BCButton">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>10</y>
     <width>121</width>
     <height>45</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>121</width>
     <height>45</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>121</width>
     <height>45</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border: 2px solid #f3f3f3;/*设置线宽*/
	background-color: rgb(237, 242, 255);/*背景颜色*/
    border-radius: 15px;/*圆角*/
    padding: 2px 8px 15px 2px;  /*针对于组合框中的文本内容*/
	text-align:bottom;
    min-width: 9em;   /*# 组合框的最小宽度*/
    /*min-height: 5em;*/

	border-style:solid;/*边框为实线型*/
	border-width:2px;/*边框宽度*/
	border-color:rgb(77, 123, 255);/*边框颜色*/

	padding-left: 10px;/*左侧边距*/
}

QPushButton:hover
{
		color:#f3f3f3;
	background-color:rgb(102,178,255); /*改变背景色*/
	border-style:inset;/*改变边框风格*/
	padding-left:6px;
	padding-top:6px;
}</string>
   </property>
   <property name="text">
    <string>获取波长</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>60</y>
     <width>611</width>
     <height>261</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox {
    border: 2px solid white;
    border-radius: 5px;
    margin-top: 1.5ex;
}
QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top left;
    padding: 0 3px;
    color: white;
    font-weight: bold;
}</string>
   </property>
   <property name="title">
    <string>光谱</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>591</width>
      <height>231</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QScrollArea" name="scrollArea_2">
       <property name="maximumSize">
        <size>
         <width>463</width>
         <height>229</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(216, 215, 222)</string>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>461</width>
          <height>227</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>全选</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2" alignment="Qt::AlignmentFlag::AlignHCenter">
        <widget class="QListWidget" name="listWidget">
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>205</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>全不选</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>330</y>
     <width>611</width>
     <height>261</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox {
    border: 2px solid white;
    border-radius: 5px;
    margin-top: 1.5ex;
}
QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top left;
    padding: 0 3px;
    color: white;
    font-weight: bold;
}</string>
   </property>
   <property name="title">
    <string>波长</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>591</width>
      <height>241</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QTableView" name="BCTable">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>220</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>220</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QTableView {
    background-color: #ffffff;
    gridline-color: #d0d0d0;
    border: 1px solid #999999;
    selection-background-color: #66aaff;
    selection-color: white;
    alternate-background-color: #f0f0f5;
}

QTableView::item {
    padding: 5px;
    font-size: 12px;
    color: #333333;
}

QTableView::item:selected {
    background-color: #66aaff;
    color: white;
}

QTableView::horizontalHeader {
    background-color: #e0e0e0;
    border-bottom: 1px solid #999999;
}

QTableView::horizontalHeader::section {
    padding: 5px;
    font-weight: bold;
    border-right: 1px solid #d0d0d0;
}

QTableView::verticalHeader {
    background-color: #e0e0e0;
    border-right: 1px solid #999999;
}

QTableView::verticalHeader::section {
    padding: 5px;
    font-weight: bold;
    border-bottom: 1px solid #d0d0d0;
}</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>261</width>
     <height>231</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox {
    border: 2px solid white;
    border-radius: 5px;
    margin-top: 1.5ex;
}
QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top left;
    padding: 0 3px;
    color: white;
    font-weight: bold;
}</string>
   </property>
   <property name="title">
    <string>地理信息</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>241</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_1_1">
       <property name="styleSheet">
        <string notr="true">QLineEdit {
    border: 2px solid #f3f3f3;/*设置线宽*/
	background-color: rgb(237, 242, 255);/*背景颜色*/
    border-radius: 15px;/*圆角*/
    padding: 2px 8px 5px 2px;  /*针对于组合框中的文本内容*/
	text-align:bottom;
    min-width: 5em;   /*# 组合框的最小宽度*/
    /*min-height: 5em;*/

	border-style:solid;/*边框为实线型*/
	border-width:2px;/*边框宽度*/
	border-color:rgb(77, 123, 255);/*边框颜色*/

	padding-left: 10px;/*左侧边距*/
}</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:700; color:#00aaff;&quot;&gt;编号&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEdit_1_2">
       <property name="styleSheet">
        <string notr="true">QLineEdit {
    border: 2px solid #f3f3f3;/*设置线宽*/
	background-color: rgb(237, 242, 255);/*背景颜色*/
    border-radius: 15px;/*圆角*/
    padding: 2px 8px 5px 2px;  /*针对于组合框中的文本内容*/
	text-align:bottom;
    min-width: 5em;   /*# 组合框的最小宽度*/
    /*min-height: 5em;*/

	border-style:solid;/*边框为实线型*/
	border-width:2px;/*边框宽度*/
	border-color:rgb(77, 123, 255);/*边框颜色*/

	padding-left: 10px;/*左侧边距*/
}</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:700; color:#00aaff;&quot;&gt;纬度&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:700; color:#00aaff;&quot;&gt;经度&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEdit_1_3">
       <property name="styleSheet">
        <string notr="true">QLineEdit {
    border: 2px solid #f3f3f3;/*设置线宽*/
	background-color: rgb(237, 242, 255);/*背景颜色*/
    border-radius: 15px;/*圆角*/
    padding: 2px 8px 5px 2px;  /*针对于组合框中的文本内容*/
	text-align:bottom;
    min-width: 5em;   /*# 组合框的最小宽度*/
    /*min-height: 5em;*/

	border-style:solid;/*边框为实线型*/
	border-width:2px;/*边框宽度*/
	border-color:rgb(77, 123, 255);/*边框颜色*/

	padding-left: 10px;/*左侧边距*/
}</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="minimumSize">
        <size>
         <width>45</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:700; color:#00aaff;&quot;&gt;巷道名称&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_1_0">
       <property name="styleSheet">
        <string notr="true">QLineEdit {
    border: 2px solid #f3f3f3;/*设置线宽*/
	background-color: rgb(237, 242, 255);/*背景颜色*/
    border-radius: 15px;/*圆角*/
    padding: 2px 8px 5px 2px;  /*针对于组合框中的文本内容*/
	text-align:bottom;
    min-width: 5em;   /*# 组合框的最小宽度*/
    /*min-height: 5em;*/

	border-style:solid;/*边框为实线型*/
	border-width:2px;/*边框宽度*/
	border-color:rgb(77, 123, 255);/*边框颜色*/

	padding-left: 10px;/*左侧边距*/
}</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:700; color:#00aaff;&quot;&gt;矿区名称&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="lineEdit_1_4">
       <property name="styleSheet">
        <string notr="true">QLineEdit {
    border: 2px solid #f3f3f3;/*设置线宽*/
	background-color: rgb(237, 242, 255);/*背景颜色*/
    border-radius: 15px;/*圆角*/
    padding: 2px 8px 5px 2px;  /*针对于组合框中的文本内容*/
	text-align:bottom;
    min-width: 5em;   /*# 组合框的最小宽度*/
    /*min-height: 5em;*/

	border-style:solid;/*边框为实线型*/
	border-width:2px;/*边框宽度*/
	border-color:rgb(77, 123, 255);/*边框颜色*/

	padding-left: 10px;/*左侧边距*/
}</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:700; color:#00aaff;&quot;&gt;长度&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="lineEdit_1_5">
       <property name="styleSheet">
        <string notr="true">QLineEdit {
    border: 2px solid #f3f3f3;/*设置线宽*/
	background-color: rgb(237, 242, 255);/*背景颜色*/
    border-radius: 15px;/*圆角*/
    padding: 2px 8px 5px 2px;  /*针对于组合框中的文本内容*/
	text-align:bottom;
    min-width: 5em;   /*# 组合框的最小宽度*/
    /*min-height: 5em;*/

	border-style:solid;/*边框为实线型*/
	border-width:2px;/*边框宽度*/
	border-color:rgb(77, 123, 255);/*边框颜色*/

	padding-left: 10px;/*左侧边距*/
}</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>261</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox {
    border: 2px solid white;
    border-radius: 5px;
    margin-top: 1.5ex;
}
QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top left;
    padding: 0 3px;
    color: white;
    font-weight: bold;
}</string>
   </property>
   <property name="title">
    <string>管理归属人</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>241</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="1" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:700; color:#00aaff;&quot;&gt;区号&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:700; color:#00aaff;&quot;&gt;名称&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_2_2">
       <property name="styleSheet">
        <string notr="true">QLineEdit {
    border: 2px solid #f3f3f3;/*设置线宽*/
	background-color: rgb(237, 242, 255);/*背景颜色*/
    border-radius: 15px;/*圆角*/
    padding: 2px 8px 5px 2px;  /*针对于组合框中的文本内容*/
	text-align:bottom;
    min-width: 5em;   /*# 组合框的最小宽度*/
    /*min-height: 5em;*/

	border-style:solid;/*边框为实线型*/
	border-width:2px;/*边框宽度*/
	border-color:rgb(77, 123, 255);/*边框颜色*/

	padding-left: 10px;/*左侧边距*/
}</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_2_1">
       <property name="styleSheet">
        <string notr="true">QLineEdit {
    border: 2px solid #f3f3f3;/*设置线宽*/
	background-color: rgb(237, 242, 255);/*背景颜色*/
    border-radius: 15px;/*圆角*/
    padding: 2px 8px 5px 2px;  /*针对于组合框中的文本内容*/
	text-align:bottom;
    min-width: 5em;   /*# 组合框的最小宽度*/
    /*min-height: 5em;*/

	border-style:solid;/*边框为实线型*/
	border-width:2px;/*边框宽度*/
	border-color:rgb(77, 123, 255);/*边框颜色*/

	padding-left: 10px;/*左侧边距*/
}</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_10">
       <property name="minimumSize">
        <size>
         <width>45</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:700; color:#00aaff;&quot;&gt;等级&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEdit_2_3">
       <property name="styleSheet">
        <string notr="true">QLineEdit {
    border: 2px solid #f3f3f3;/*设置线宽*/
	background-color: rgb(237, 242, 255);/*背景颜色*/
    border-radius: 15px;/*圆角*/
    padding: 2px 8px 5px 2px;  /*针对于组合框中的文本内容*/
	text-align:bottom;
    min-width: 5em;   /*# 组合框的最小宽度*/
    /*min-height: 5em;*/

	border-style:solid;/*边框为实线型*/
	border-width:2px;/*边框宽度*/
	border-color:rgb(77, 123, 255);/*边框颜色*/

	padding-left: 10px;/*左侧边距*/
}</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>480</y>
     <width>261</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox {
    border: 2px solid white;
    border-radius: 5px;
    margin-top: 1.5ex;
}
QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top left;
    padding: 0 3px;
    color: white;
    font-weight: bold;
}</string>
   </property>
   <property name="title">
    <string>主机设置</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>241</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_6">
     <item row="0" column="0">
      <widget class="QLabel" name="label_12">
       <property name="minimumSize">
        <size>
         <width>45</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700; color:#00aaff;&quot;&gt;远程IP地址&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_3_1">
       <property name="styleSheet">
        <string notr="true">QLineEdit {
    border: 2px solid #f3f3f3;/*设置线宽*/
	background-color: rgb(237, 242, 255);/*背景颜色*/
    border-radius: 15px;/*圆角*/
    padding: 2px 8px 5px 2px;  /*针对于组合框中的文本内容*/
	text-align:bottom;
    min-width: 5em;   /*# 组合框的最小宽度*/
    /*min-height: 5em;*/

	border-style:solid;/*边框为实线型*/
	border-width:2px;/*边框宽度*/
	border-color:rgb(77, 123, 255);/*边框颜色*/

	padding-left: 10px;/*左侧边距*/
}</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" alignment="Qt::AlignmentFlag::AlignHCenter">
      <widget class="QPushButton" name="SetButton">
       <property name="minimumSize">
        <size>
         <width>110</width>
         <height>45</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>121</width>
         <height>45</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    border: 2px solid #f3f3f3;/*设置线宽*/
	background-color: rgb(237, 242, 255);/*背景颜色*/
    border-radius: 15px;/*圆角*/
    padding: 2px 8px 15px 2px;  /*针对于组合框中的文本内容*/
	text-align:bottom;
    min-width: 8em;   /*# 组合框的最小宽度*/
    /*min-height: 5em;*/

	border-style:solid;/*边框为实线型*/
	border-width:2px;/*边框宽度*/
	border-color:rgb(77, 123, 255);/*边框颜色*/

	padding-left: 10px;/*左侧边距*/
}

QPushButton:hover
{
		color:#f3f3f3;
	background-color:rgb(102,178,255); /*改变背景色*/
	border-style:inset;/*改变边框风格*/
	padding-left:6px;
	padding-top:6px;
}</string>
       </property>
       <property name="text">
        <string>设置</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <zorder>bg_label</zorder>
  <zorder>SaveButton</zorder>
  <zorder>label_16</zorder>
  <zorder>GPButton</zorder>
  <zorder>BCButton</zorder>
  <zorder>groupBox</zorder>
  <zorder>groupBox_2</zorder>
  <zorder>groupBox_3</zorder>
  <zorder>groupBox_4</zorder>
  <zorder>groupBox_5</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
